### ISSUE TITLE
Not Working in Redmi k20 Pro (Android 10 - API 29)

### ISSUE THREAD

**Issue by iamkdblue** [iFr0z/WorkManagerNotification#1]
I am using Redmi k20 Pro (Android 10), notification is not receiving when the app is killed.
Any solution?

**Comment by iFr0z** [comment]
Are you using the latest version WorkManager?

**Comment by iamkdblue** [comment]
Yes, I am using `implementation 'androidx.work:work-runtime:2.4.0-alpha03' 
`

**Comment by acsbendi** [comment]
@iamkdblue WorkManager does not work reliably on certain devices. You should check https://dontkillmyapp.com/ for more information.
